openehr-rm/src/main/java/com/nedap/archie/rm/datavalues/quantity/DvScale.java

```diff
@@ -18,7 +18,7 @@
         "value",
         "symbol"
 })
-public class DvScale extends DvOrdered<DvScale, DvScale> implements SingleValuedDataValue<Double>, Comparable<DvScale> {
+public class DvScale extends DvOrdered<DvScale> implements SingleValuedDataValue<Double>, Comparable<DvScale> {
```

Comment:
There's an additional Comparable<DvScale> here, looks like that can be removed as well, because of the definition in the parent being already there.